content/en/shift_left.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. Although originally used to refer to the process of testing early, Shift Left can now also be applied to other aspects of software development and DevOps, such as Security and Deployment. 
```

Comment:
This is great and very easy to understand! It'd be great if we could explain why it's called "shift left." Not everyone may be familiar with the concept of moving code from left to right so it may not be intuitive. I assume it's because we are moving test and security further up (or left) in the process.